### DISCUSSION TITLE
Is there an alternative for ReactiveRecyclerViewViewHolder<TViewModel> set as obsolete  in ReactiveUI.AndroidX?

### DISCUSSION THREAD

**Discussion by LannoySeb** [reactiveui/ReactiveUI#3673] [Q&A]
This year, the constructor of `ReactiveRecyclerViewViewHolder<TViewModel>` has been marked as obsolete in the `ReactiveUI.AndroidX` package. Here is the pull request that merged the change: https://github.com/reactiveui/ReactiveUI/pull/3579.

Unfortunately for me, I was using it in a project with strict policies regarding warnings. Therefore, in order to remove the warning indicating that this constructor is obsolete, I searched for a non-obsolete alternative, but without success.

Is there an alternative to the obsolete `ReactiveRecyclerViewViewHolder<TViewModel>` from the `ReactiveUI.AndroidX` package, or is this still the constructor that we are supposed to use?

**Comment by ChrisPulman** [comment]
Hi, thank you for reporting this, I will check it, as some code is reused in some places, I may have marked it for one implementation which would then be applied to both.
Many apologies for any inconvenience caused.

**Reply by LannoySeb** [reply]
There is no problem, thank you for your time.
